app/templates/components/nypr-svg/trending.hbs

```diff
@@ -0,0 +1,3 @@
+<svg class={{className}} width="11px" height="20px" viewBox="0 0 11 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <path d="M10.1506348,11.4862258 C8.92127838,9.90159451 8.89765455,8.09379835 9.53985567,6.49746985 C8.17884801,7.83049589 7.41256446,9.4786592 7.94765561,11.1428775 C8.01554544,11.3545745 8.05912337,11.5619138 8.09100407,11.7736108 L8.10316002,12.0288857 L8.02724268,12.0568673 L7.7040779,11.3183361 C7.2449041,10.0614567 7.17724362,8.73622883 7.62816055,7.37522116 C8.52609533,4.70939844 7.3529315,2.09908024 5.21371369,0 C6.19949241,2.53807058 6.1077494,5.42384719 4.11233877,7.97820215 C3.72908231,8.46512821 3.41394505,8.96008179 3.16256918,9.4667326 L3.01073449,9.80182397 L3.02289044,9.71420939 C3.42587164,7.99815625 2.54789096,6.44150661 1.08711277,5.26834277 C1.84536881,6.78875393 1.96119437,8.59288036 0.859590088,10.3089335 C-0.0624272369,12.0807205 -0.297977434,14.28003 0.424498828,15.9123676 C1.55408473,18.2710806 3.05477114,19.3566299 5.09421841,19.4123638 C5.38940157,19.4162628 5.6928416,19.4045656 6.00430914,19.364428 C7.66852748,19.1848411 9.74788297,18.3710805 10.4182951,16.4433304 C11.0806797,14.7905799 11.3960463,13.0981506 10.1506348,11.4862258 Z" id="trending" stroke="none" fill="#FBE01A" fill-rule="nonzero"></path>
```

Comment:
shouldn't you get rid of the `fill` attribute so the color can be controlled via CSS?